PS4k/src/com/vitco/core/data/VoxelData.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.Set;
```

Comment:
Pretty sure this is not used anymore. Always have a look at your commits, sometimes they include weirdness like this.